src/Microsoft.Health.Fhir.TemplateManagement/Models/OCIArtifactLayer.cs

```diff
@@ -25,17 +25,17 @@ public class OCIArtifactLayer
         // Content of the layer.
         public virtual byte[] Content { get; set; }
 
-        public void WriteToFolder(string directory)
+        public void WriteToFolder(string path)
         {
-            EnsureArg.IsNotNullOrEmpty(directory, nameof(directory));
+            EnsureArg.IsNotNullOrEmpty(path, nameof(path));
 
             if (Content == null)
```

Comment:
Is it possible to be an empty file there?

Reply:
Content for empty file will be empty byte list but not null.

Reply:
When will the Content be null? Can we treat it as empty?

Reply:
Content will be null when new an empty layer object "new OCIArtifactLayer()", but we hope this layer will be be write to the folder.